docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md

```diff
@@ -0,0 +1,107 @@
+
+- Start and re-register Microsoft Installer service. It requires some work.
+
+    1) Press *Win + R* and enter **services.msc**.
+    2) Find in the list and double click on *Windows Installer*.
+    3) Hit *Start* button under *Service status* and hit *OK*. If the service status is **running**, you should click on *Stop* first and then hit *Start*. 
+    4) Press *Win + R*, type and enter ***msiexec /unregister*** and hit *Enter*.
+    5) Press *Win + R* again, type and enter ***msiexec /regserver*** and hit *Enter*
+
+- Reboot the PC and start the installation all over again. Sometimes that's enough to fix the problem.
+
+### Error 1602: Canceled by user {#error-1602}
+
+If you got this error code, chances are you have interrupted the installation process manually in one way or another. What you can do is:
+
+- Don't close the installer window. When the installation is complete, it will close automatically.
+
+- If a dialogue window pops up during the installation, hit "Yes" to grant the installer the required permissions. Clicking on "No" will cancel he installation.
+
+- Don't start other processes while the installation process is going on.
+
+### Error 1603: Fatal error during installation {#error-1603}
+
+The error sounds scarier than it actually is. In reality, this is a rather generic error that can have many different causes, and some of them are easily fixed. Try the following solutions:
+
+- Press the *Win* key, search for *Command Prompt* and run it. There, type in `sfc /scannow` and press *Enter*.
+
+- Choose a different installation folder. It is possible that the current installation folder has some access restrictions. Also make sure you don't select an external drive, a virtual drive, etc.
+
+- Uninstall AdGuard VPN using our special [uninstall tool](../../installation#advanced) and then repeat the installation.
+
+- Start and re-register Microsoft Installer service. It requires some work.
+
+    1) Press *Win + R* and enter ***services.msc***.
+    2) Find in the list and double click on *Windows Installer*.
+    3) Hit *Start* button under *Service status* and hit *OK*. If the service status is **running**, you should click on *Stop* first and then hit *Start*. 
+    4) Press *Win + R*, type and enter ***msiexec /unregister*** and hit *Enter*.
+    5) Press *Win + R* again, type and enter ***msiexec /regserver*** and hit *Enter*
+
+- Acquire full permissions on the drive for installation. It is possible that the error 1603 occurs because you don’t have full permissions on the file location. It's also not as easy as some of the other solutions:
+
+    1) Open *File Explorer*, right click on the drive containing the installation location and select *Properties*.
+    2) Go to *Security* tab and click on *Edit*.
+    3) Single click on *SYSTEM* and ensure that the *Allow* box of every item in *Permissions for SYSTEM* is checked (if it is checkable). Do the same check for *Administrators*.
```

Comment:
Click *SYSTEM*

Reply:
We don't see how it looks in the interface, I suppose there is a reason why it deliberately says "single", therefore I suggest "Single-click _SYSTEM_"